File: Java/q1044.java
// Múltiplos

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        int b = input.nextInt();
        
        if (a > b){
            int aux = b;
            b = a;
            a = aux;
        }
        if (b%a != 0){
            System.out.println("Nao sao Multiplos");
        } else {
            System.out.println("Sao Multiplos");
        }
    }
}

File: Cpp/q1044.cpp
#include<iostream>

using namespace std;

int main(){

  int n1, n2, divisor1, divisor2;
  cin >> n1 >> n2;

  divisor1 = n2%n1;
  divisor2 = n1%n2;

  if(divisor1 == 0 || divisor2 == 0){
      cout << "Sao Multiplos" << endl;
  } else {
      cout << "Nao sao Multiplos" << endl;
  }
}